sqfentity/lib/sqfentity.dart

```diff
@@ -292,6 +292,12 @@ class SqfEntityProvider extends SqfEntityModelBase {
     try {
       /// Leave it in this format for Throw to stay in this catch
       final res = await updateOrThrow(obj);
+
+      if (_dbModel!.postSaveAction != null) {
+        final record = obj.toMap(forQuery: true);
+        await _dbModel!.postSaveAction!(_tableName!, record, 'INSERT');

```

Comment:
This should be update 
```suggestion
        await _dbModel!.postSaveAction!(_tableName!, record, 'UPDATE');

```

Reply:
Haha, copy and paste mistake. Resolved.